src/components/Button/index.tsx

```diff
@@ -80,6 +83,7 @@ export const Button = forwardRef(
       disabled,
       label,
       isActive,
+      isLabelUpperCase = true,
```

Comment:
i'm not entire sure the default value here should be true, maybe a different wording on the prop name.
lets see what the @reywright thinks.

looks good to me otherwise.

Reply:
Our design system mirrors designs intentions, so it just depends on what they think button text should look like by default.  I think `isUpperCase` would be a nice prop here.